packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -93,10 +93,33 @@ export class SearchFilterTransform {
     const filterStrings: string[] = [];
     facets.forEach(facet => {
       const field = get(facet, 'field', '');
-      const results = get(facet, 'results', []);
-      const keys = this.quoteSelectedFacets(results, 'key');
-      if (keys.length) {
-        filterStrings.push(keys.map(key => `${escapeFieldName(field)}::${key}`).join('|'));
+      const results: SelectableQueryTermAggregationResult[] | never[] = get(facet, 'results', []);
```

Comment:
Why `never`?

Reply:
Also, let's get rid of `get()` here:
```suggestion
      const results: SelectableQueryTermAggregationResult[] | never[] = facet?.results || [];
```

That also makes TS work better.

Reply:
👍🏻 turns out I could get rid of the `never` typing without an error popping up

Reply:
👍🏻 turns out I could get rid of the `never` typing without an error popping up